File: Flaadestation.API/Flaadestation.Repository/Database/Entities/Job.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Flaadestation.Repository.Database.Entities
{
    public class Job
    {
        public Guid JobId { get; set; }
        public string Title { get; set; } = string.Empty;
        public string Description { get; set; } = string.Empty;
        public DateTime ScheduledStart { get; set; }
        public DateTime ScheduledEnd { get; set; }
        public Guid CompanyId { get; set; }
        public Guid StorageId { get; set; }
        public Guid AddressId { get; set; }

        public Company? Company { get; set; }
        public Storage? Storage { get; set; }
        public List<Customer> Customers { get; set; } = [];
    }
}

File: Flaadestation.Dashboard/src/Models/Job.ts
import type { Address } from "./Address";
import type { Customer } from "./Customer";

export interface Job {
    jobId: string;
    title: string;
    description: string;
    addressId: string;
    address: Address;
    scheduledStart: string;
    scheduledEnd: string;
    customers: Customer[];
    storage: Storage;
}

export class JobModel {
    job: Job

    constructor(job: Job) {
        this.job = job
    }

    get data(): Job {
        return this.job
    }
}